Tweaks to wording in the docs (#927)
Qiskit/qiskit-ibm-runtime: docs/compare.rst

How do Qiskit Runtime primitives differ from backend.run?
=========================================================

There are two methods for accessing IBM Quantum systems.  First, the
`qiskit-ibm-provider` package provides the ``backend.run()`` interface,
allowing direct access to IBM Quantum systems with no pre- or post-processing
involved.  This level of access is suitable for those users who want precise
control over circuit execution and result processing.  This level of access
is needed for those looking to work at the level Kernel developer developing,
for example, circuit optimization routines, error mitigation techniques, or
characterizing quantum systems.

> In contrast, the Qiskit Runtime is designed to streamline the construction

Reviewer: ```suggestion
In contrast, Qiskit Runtime is designed to streamline algorithm and application construction
```

> of algorithms and applications by removing the need for users to understand

Reviewer: ```suggestion
 by removing the need for users to understand
```

> technical hardware and low-level software details.  Advanced processing technics

Reviewer 2: ```suggestion
technical hardware and low-level software details.  Advanced processing techniques
```

for error suppression and mitigation are automtically applied, giving users
high-fidelity results without the burdeen of haivng to code these routines

> themselves.  The inclusion of Sessions within the Qiskit runtime allows users

Reviewer: ```suggestion
themselves.  The inclusion of sessions within Qiskit Runtime allows users
```

> to run iterative algorithm circuits back to back, or batch collections of circuits,

Reviewer: ```suggestion
to run iterative algorithm circuits back to back, or batch collections of circuits
```

> without having to re-queue each job.  This results in more efficient utilization

Reviewer: ```suggestion
without having to re-queue each job.  This results in more efficient quantum processor utilization
```

> of the qunatum processor, and reduces the total amount of time users spend running

Reviewer: ```suggestion
 and reduces the total amount of time users spend running
```
